actors/miner/src/state.rs

```diff
@@ -553,6 +553,45 @@ impl State {
         Ok(())
     }
 
+    pub fn assign_sectors_to_deadline<BS: Blockstore>(
```

Comment:
I think this is fine, it roughly matches `assign_sectors_to_deadlines`, but I'd appreciate more expert eyes on this